lane_input lane_input

# obstacle data from perception fusion
obstacle_input obs_input

# action planning
action_plan ActionPlanning_result

# trajectory planning
trajectory_plan TrajPlanning_result

# velocity planning
velocity_plan VelPlanning
